### ISSUE TITLE
Error saving structure

### ISSUE THREAD

**Issue by escowles** [pulibrary/figgy#4206]
Tom V. reported a problem saving structure on this resource: https://figgy.princeton.edu/catalog/df081902-402c-419d-b6a1-9b9c5863da3f — no error message was displayed, but the structure did not save successfully.

I found this error in DataDog:
```
{
	"id": "AQAAAXQL1FV4SAq8sgAAAABBWFFMMUZ2U2laSDhtX3FPQlFBeg",
	"content": {
		"timestamp": "2020-08-20T12:24:59.000Z",
		"tags": [
			"application:nginxplus",
			"environment:production",
			"filename:access.log",
			"service:nginxplus",
			"source:nginxplus",
			"sourcecategory:http_web_access",
			"type:adc"
		],
		"host": "lib-adc1",
		"service": "nginxplus",
		"message": "172.20.207.252 - - [20/Aug/2020:12:24:59 +0000] \"PUT /concern/scanned_resources/df081902-402c-419d-b6a1-9b9c5863da3f HTTP/2.0\" 400 0 \"https://figgy.princeton.edu/concern/scanned_resources/df081902-402c-419d-b6a1-9b9c5863da3f/structure\" \"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:79.0) Gecko/20100101 Firefox/79.0\" \"-\"",
		"attributes": {
			"http": {
				"referer": "https://figgy.princeton.edu/concern/scanned_resources/df081902-402c-419d-b6a1-9b9c5863da3f/structure",
				"status_code": 400,
				"url_details": {
					"path": "/concern/scanned_resources/df081902-402c-419d-b6a1-9b9c5863da3f"
				},
				"method": "PUT",
				"useragent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:79.0) Gecko/20100101 Firefox/79.0",
				"useragent_details": {
					"browser": {
						"family": "Firefox",
						"major": "79",
						"minor": "0"
					},
					"os": {
						"major": "10",
						"minor": "15",
						"family": "Mac OS X"
					},
					"device": {
						"family": "Other",
						"category": "Desktop"
					}
				},
				"url": "/concern/scanned_resources/df081902-402c-419d-b6a1-9b9c5863da3f",
				"status_category": "warning",
				"version": "2.0"
			},
			"network": {
				"client": {
					"ip": "172.20.207.252"
				},
				"bytes_written": 0
			},
			"date_access": 1597926299000
		}
	}
}
```

**Comment by tpendragon** [comment]
https://figgy.princeton.edu/concern/scanned_resources/df081902-402c-419d-b6a1-9b9c5863da3f/structure has structure now. A fluke?

**Comment by tpendragon** [comment]
Tom reported that @escowles removed some extra files and after that it worked. Maybe @escowles can share what he did? In any case, closing this.

**Comment by escowles** [comment]
@tpendragon Here's what I did:
1. Went to Figgy to see if I could complete the item, and saw some weird behavior (it went to the edit form instead of updating the resource
2. Opened a Figgy console and tried to complete the item, and when validating the state update, got an error that certain items in the `member_ids` didn't exist
3. I removed the items that didn't exist, and then I was able to complete the item.
